src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-styleslistitem.js

```diff
@@ -18,6 +18,7 @@ export default class EzBtnStylesListItem extends AlloyEditor.ButtonStylesListIte
                 className={className}
                 dangerouslySetInnerHTML={{ __html: this._preview }}
                 onClick={() => {
+                    this.clearEzAttributes();
```

Comment:
Now when you click on the same item (custom block style) it will not remove the style, as it does in other elements.

Reply:
@dew326 Resolved in https://github.com/ezsystems/ezplatform-admin-ui/pull/1217/commits/b7a1abda55c4f5680717e612d9dca7a8d7d0d72d